libs/libxx/libcxxabi.defs

```diff
@@ -0,0 +1,62 @@
+############################################################################
+# libs/libxx/libcxxabi.defs
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.  The
+# ASF licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+###########################################################################
+
+include $(TOPDIR)/Make.defs
+
+VERSION=12.0.0
+
+libcxxabi-$(VERSION).src.tar.xz:
+	$(Q) curl -O -L https://github.com/llvm/llvm-project/releases/download/llvmorg-$(VERSION)/libcxxabi-$(VERSION).src.tar.xz
+
+libcxxabi: libcxxabi-$(VERSION).src.tar.xz
+	$(Q) tar -xf libcxxabi-$(VERSION).src.tar.xz
+	$(Q) mv libcxxabi-$(VERSION).src libcxxabi
+
+$(TOPDIR)/include/libcxxabi: libcxxabi
+	$(Q) $(DIRLINK) $(CURDIR)/libcxxabi/include $(TOPDIR)/include/libcxxabi
+
+dirlinks:: $(TOPDIR)/include/libcxxabi
```

Comment:
let's keep libcxxabi internally

Reply:
@xiaoxiang781216 since #4369 is merged, this also needs to go to `context` (this branch needs to be rebase before to avoid a build error.)

Reply:
Yes, you are right.